app/src/main/java/com/example/playlistmaker/search/ui/view/SearchFragment.kt

```diff
@@ -131,7 +131,7 @@ class SearchActivity : AppCompatActivity() {
         }
 
         searchEditTxt.setOnEditorActionListener { _, actionId, _ ->
-            if (actionId == EditorInfo.IME_ACTION_DONE && !searchEditTxt.text.toString().isNullOrEmpty()/*!savedSearchEditText.isNullOrEmpty()*/) {
+            if (actionId == EditorInfo.IME_ACTION_DONE && !searchEditTxt.text.toString().isNullOrEmpty()) {
                 viewModel.searchDebounce(searchEditTxt.text.toString())
                 true
             }
```

Comment:
🍏 true никак не будет использоваться, если не обернут false в ветку else
```suggestion
            } else {
            false 
            }
```